### ISSUE TITLE
Unable to login without resetting password.

### ISSUE THREAD

**Issue by Versatilus** [processing/p5.js-web-editor#1221]
- Found a bug

#### Details about the bug: 
I use a simple Python program to generate passwords. The passwords were originally 48 characters long and could contain any printable ASCII character. I was successfully able to sign up with one of these passwords. However, when I log out and try to log back in the password is not accepted. When that failed the first time I restricted the list of possible characters and generated a new password of the same length. This one failed with the same issue.

The original failed password was `zFA<Xte>(ECS)6c#41L.P%700V4A.qEkb1 H @E@7N/*)ZWv`.

I'm assuming the passwords were too long, though there still might be an invalid character. It would be helpful if the sign up & login user interfaces displayed more feedback on what constitutes a valid password in terms of allowable characters and password length.

I am using Google Chrome Version 78.0.3904.108 (Official Build) (64-bit) on Windows 10.

**Comment by catarak** [comment]
Thanks for reporting! I just tried registering with that exact password, then logging out, and then logging back in again, and I was able to get it to work. Maybe there's something else going on?

**Comment by Versatilus** [comment]
This is something very weird. I just tried a generic short password with only lowercase letters and I'm still having the same problem. Is my username `Eric_Lewis_Paulson` causing problems? I've tried logging in with both the username and with my e-mail address, all without success.

I also tried logging in with a recent version of Firefox and I still can't log back in without resetting the password.
